### PR TITLE
expand docs re: maths & multiplication

### PR DESCRIPTION
**Pull request by xwvvvvwx**

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -20,18 +20,32 @@ Dai has three different numerical units: `wad`, `ray` and `rad`
 
 - `wad`: fixed point decimal with 18 decimals (for basic quantities)
```
Comment (asymmetric) [inline DEVELOPING.md]: e.g. balances

File: DEVELOPING.md [hunk 2]
```diff
@@ -20,18 +20,32 @@ Dai has three different numerical units: `wad`, `ray` and `rad`
...
 
 Generally, `wad` should be used additively and `ray` should be used
-multiplicatively. It usually doesn't make sense to multiply a wad by a
-wad.
+multiplicatively. It usually doesn't make sense to multiply a `wad` by a
+`wad`.
 
 The base of `ray` is `ONE = 10 ** 27`.
 
+A good explanation of fixed point arithmetic can be found at [wikipedia](https://en.wikipedia.org/wiki/Fixed-point_arithmetic).
```
Comment (asymmetric) [inline DEVELOPING.md]: I would capitalize Wikipedia.

File: DEVELOPING.md [hunk 3]
```diff
@@ -20,18 +20,32 @@ Dai has three different numerical units: `wad`, `ray` and `rad`
...
 
 `wad` and `ray` units will be familiar from SCD. `rad` is a new unit and
-exists to prevent precision loss in the core CDP engine. Instead of
-using fixed point multiplication (e.g. `rmul`), integer multiplication is used.
+exists to prevent precision loss in the core CDP engine.
 
 Generally, `wad` should be used additively and `ray` should be used
-multiplicatively. It usually doesn't make sense to multiply a wad by a
-wad.
+multiplicatively. It usually doesn't make sense to multiply a `wad` by a
```
Comment (asymmetric) [inline DEVELOPING.md]: As discussed, I'd move this to the next section.

File: DEVELOPING.md (hunk 2 above)
Comment (xwvvvvwx) [reply DEVELOPING.md]: done

File: DEVELOPING.md (hunk 3 above)
Comment (xwvvvvwx) [reply DEVELOPING.md]: done

File: DEVELOPING.md [hunk 4]
```diff
@@ -18,20 +18,34 @@ previous iteration of Dai was called Single Collateral Dai (SCD), or
...
+
+A good explanation of fixed point arithmetic can be found at [Wikipedia](https://en.wikipedia.org/wiki/Fixed-point_arithmetic).
+
+## Multiplication
 
 Generally, `wad` should be used additively and `ray` should be used
-multiplicatively. It usually doesn't make sense to multiply a wad by a
-wad.
+multiplicatively. It usually doesn't make sense to multiply a `wad` by a
+`wad` (or a `rad` by a `rad`).
```
Comment (rainbreak) [inline DEVELOPING.md]: No, you can multiply a rad by a rad just fine, e.g. a price multiplied by a ratio.

**Pull request merged**